dev/statement_serializator.h

```diff
@@ -812,6 +812,16 @@ namespace sqlite_orm {
             }
         };
 
+        template<class T>
+        struct statement_serializator<generated_always_as_t<T>, void> {
+            using statement_type = generated_always_as_t<T>;
+
+            template<class C>
+            std::string operator()(const statement_type& c, const C& context) const {
+                return static_cast<std::string>(c) + " " + serialize(c.expression, context);
```

Comment:
it would be great to add some `statement_serializator` tests for `generated_always_as_t`